From 1914c5d864de8081af5f9d372c1489fcec61d785 Mon Sep 17 00:00:00 2001
From: Asa Jones <ajones@oneclearchoice.com>
Date: Thu, 6 Apr 2017 00:27:48 -0400
Subject: [PATCH] create version json

---
 version.json | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 version.json

diff --git a/version.json b/version.json
new file mode 100644
index 0000000..bd88ffe
--- /dev/null
+++ b/version.json
@@ -0,0 +1,3 @@
+{
+  "version": "3.5.5"
+}
-- 
1.9.1

